src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -108,16 +109,16 @@ public class ScriptHost : JobHost
 
         public event EventHandler IsPrimaryChanged;
 
-        public string InstanceId
+        public string ScriptHostInstanceId
```

Comment:
You should really call this InstanceId - it's an instance ID on type ScriptHost - no need to call it ScriptHost.ScriptHostInstanceId, since it's the instance ID of the current type.

Reply:
Yes. Makes sense. I removed constant that we called InstanceId. Will update